tools/topology/sof-cnl-rt5682-sdw2.m4

```diff
@@ -0,0 +1,143 @@
+#
+# Topology for WHL with rt5682 on link2
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`alh.m4')
+include(`muxdemux.m4')
```

Comment:
We probably don't need muxdemux.m4

Reply:
fixed and updated to use new HDAudio DAI config